doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
```

Comment:
Is this line necessary? I am not entirely sure I understand what it is supposed to say, apart from the same thing the next sentence says.